app/src/models/status.ts

```diff
@@ -8,27 +8,27 @@ import { ConflictFileStatus } from './conflicts'
  */
 export enum GitStatusEntry {
   // M
-  Modified,
+  Modified = 'M',
   // A
-  Added,
+  Added = 'A',
   // D
-  Deleted,
+  Deleted = 'D',
   // R
-  Renamed,
+  Renamed = 'R',
   // C
-  Copied,
+  Copied = 'C',
   // .
-  Unchanged,
+  Unchanged = '.',
   // ?
-  Untracked,
+  Untracked = '?',
   // !
-  Ignored,
+  Ignored = '!',
```

Comment:
:art: now that these are string enums the code comments are unnecessary - feel free to tidy them up while we're in here